### PR TITLE
🥑 Add one one liner description to resource cards

### LINKED ISSUE #342: Add one-liner preview to resource cards
We want to add a preview of the description of the resource to our resource cards according to the [design](https://www.figma.com/file/JDI6BvoiJBG5JtGHisQOMy/YMCA-NAWC-Sp21?node-id=129%3A5349&frame-preset-name=Desktop).  This will just be taking as much of the description as possible that will fit on one line in the card, and then adding an ellipsis to show there's morel.

The code will probably go in [ResourcePreview](https://github.com/hack4impact-uiuc/ymca/blob/master/client/src/components/ResourcePreview.jsx).

### PR DESCRIPTION
**Pull request by lucianatl**
## Description

- Added one liner description to resource cards with ellipses when there is overflow
- Fixed resourceGrid css so that cards don't become massive

Fixes #342 

## Screenshots
image

### CODE REVIEW

File: client/src/components/ResourcePreview.jsx
Comment (jackieo5023) [inline client/src/components/ResourcePreview.jsx:91]: I think a comma was accidentally copied at the end of this line 😂 can it be removed?

File: client/src/css/ResourcePreview.css
Comment (jackieo5023) [inline client/src/css/ResourcePreview.css:28]: nit - can this be named something like `resource-preview-description`? I believe css can get weird with it kind of being global, so it's best to be descriptive with what component the CSS is relevant for

File: client/src/components/ResourcePreview.jsx
Comment (jackieo5023) [inline client/src/components/ResourcePreview.jsx:64]: Make sure to address this lint comment :)

File: client/src/components/ResourcePreview.jsx
Comment (jackieo5023) [inline client/src/components/ResourcePreview.jsx:91]: See my comment on the CSS file - just make sure to change the className here as well

**Review by jackieo5023 — APPROVED** [review]
Looks great 😄 just a few nits but after addressing those and getting a second approval, feel free to merge!

**Review by nkonjeti — APPROVED** [review]
Looks good!

**Review by ayanmallik — APPROVED** [review]
Super clean implementation

**Pull request merged**
